build.gradle.kts

```diff
@@ -64,12 +64,12 @@ dependencies {
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
     implementation("io.github.microutils:kotlin-logging-jvm:3.0.4")
     implementation("org.apache.commons:commons-lang3:3.12.0")
-    implementation("org.mapstruct:mapstruct:1.5.3.Final")
-    kapt("org.mapstruct:mapstruct-processor:1.5.3.Final")
+    implementation("org.mapstruct:mapstruct:1.5.5.Final")
+    kapt("org.mapstruct:mapstruct-processor:1.5.5.Final")
 
     // internal libs
-    implementation("uk.gov.dluhc:logging-library:2.3.1")
-    implementation("uk.gov.dluhc:messaging-support-library:1.1.0")
+    implementation("uk.gov.dluhc:logging-library:3.0.3")
+    implementation("uk.gov.dluhc:messaging-support-library:2.1.0")
 
     // api
     implementation("org.springframework.boot:spring-boot-starter-actuator")
```

Comment:
A few lines down (can't comment on that line) there is an openapi-ui dependency, we should remove it and replace with:
```
  implementation("io.swagger.core.v3:swagger-annotations:2.2.7")
  implementation("org.springframework:spring-webmvc")
```

I've now updated the guidance page to include that step as it's a recently reported vulnerability